library/src/main/res/values/library_mpandroidchart_strings

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_mpandroidchart"></string>
```

Comment:
it should say define_int_mpandroidchart

(The int defines that this library is inside of the AboutLibraries library and should not be displayed automatically if the xml is found, only if the user chooses to show it)